apm-server.ssl.yml

```diff
@@ -0,0 +1,872 @@
+################### APM Server Configuration #########################
```

Comment:
out of curiosity, what is this one for?

Reply:
hah! You caught my, that was my config I played around with during dev, will remove.